doc/examples/widgets/lib/main.dart

```diff
@@ -134,8 +135,9 @@ void main() async {
             animation: _animation,
             playing: ctx.boolProperty('playing', true),
             anchor: parseAnchor(
-              ctx.listProperty('anchor', 'Anchor.center', anchorOptions),
-            ),
+                  ctx.listProperty('anchor', 'Anchor.center', anchorOptions),
+                ) ??
+                Anchor.topLeft,
```

Comment:
I think parseAnchor should just throw instead of returning null

Reply:
#607